openpype/modules/interfaces.py

```diff
@@ -397,8 +412,8 @@ def set_service_idle_icon(self):
 class ISettingsChangeListener(OpenPypeInterface):
     """Module has plugin paths to return.
 
-    Expected result is dictionary with keys "publish", "create", "load" or
-    "actions" and values as list or string.
+    Expected result is dictionary with keys "publish", "create", "load",
+    "actions" or "inventory" and values as list or string.
     {
```

Comment:
I added it to the docstring here, but I suspect this docstring was already wrong to begin with hehe.

Reply:
Yes, looks like the docstring was copied from `IPluginPath`.

Reply:
Any decent replacement docstring you could write up for `ISettingsChangeListener`? :) I can include it here if that makes sense.

Reply:
Not decent at all, but don't have time for more.
```
"""Module want to catch settings changes.

Only changes that happened in current process are propagated. Changes made in other processes, or machines won't  trigger the callbacks.
```